api/src/paths/project/{projectId}/attachments/security/add.ts

```diff
@@ -54,11 +54,20 @@ POST.apiDoc = {
                 minimum: 1
               }
             },
-            attachment_ids: {
+            attachments: {
               type: 'array',
               items: {
-                type: 'number',
-                minimum: 1
+                type: 'object',
+                minimum: 1,
```

Comment:
nit: remove, I dont think this is doing anything at this level, might just be a remnant from the previous version

Reply:
removed